src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.keys(robot);
+  const newRobot = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
+    if (newRobot.hasOwnProperty(robot[robotKeys[i]])) {
+      return null;
+    }
+    newRobot[robot[robotKeys[i]]] = robotKeys[i];
+  }
```

Comment:
Please use variable for `robotKeys[i]`

Reply:
> Please use variable for `robotKeys[i]`

Can you show me example what you exactly mean?
 I have variable before in code const robotKeys = Object.keys(robot);

Reply:
1. Kacper means you need to create variable with `robotKeys[i]` and use it inside loop, so your code would be easier to read
2. I think you can use `for in` loop through object insted of creating array with robot keys to simplify your solution